scripts/batch-score.coffee

```diff
@@ -0,0 +1,50 @@
+# Description:
+#   Script for batch wise score.
+#
+# Commands:
+#   hubot score bxx
+# 
+# Author:
+#   aseem09
+
+cron = require('node-cron')
+module.exports = (robot) ->
+
+  listOfUsersWithCount = (year) ->
+  
+    sorted = []
+    for own key, user of robot.brain.data.users
+
+      temp2=user[4]
+      user_year=temp2.split('')
+      temp3 = parseInt( user_year[0], 10 );
+
+      if `temp3 == year` > 0
+        sorted.push [user.name, user.msgcount]
```

Comment:
Why do you need `msgcount` of the users?

Reply:
Forgot to change it. I'll do the necessary changes